test/integration/integration.test.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+// override Smartcar's browser lint rules for Jest tests
+// note that Jest ships with jsdom so window is loaded globally in Jest tests
```

Comment:
Looked into mocking window using Jest. Not sure if helpful but you can mock `window` by mocking `origin`.

So, to mock `window.location.origin` you could do:
```js
global.location = {
  origin: 'mock origin',
};
```
When the tests are run, `window.location.origin` will be replaced by `'mock origin'`. See https://github.com/smartcar/developer-client/pull/53 for how I set up mocks for `window.location`.

Reply:
Does this allow setting different origins in different tests? As far as I can tell it seems to establish the mocked location once at the beginning of all tests?